docs/podman-containers-cleanup.1.md

```diff
@@ -0,0 +1,38 @@
+% podman-container-cleanup "1"
+
+## NAME
+podman\-container\-cleanup - Cleanup Container storage and networks
+
+## SYNOPSIS
+**podman** **container-cleanup** [*options* [...]] container
+
+## DESCRIPTION
```

Comment:
After reading this, I'm not sure what "clean up" is meant to do.  Can you add a few details?  Perhaps "Cleans the container by removing all mountpoints and networks."

Reply:
I like the added verbiage, tyvm!